--- a/package/firmware/linux-firmware/mediatek.mk	2025-03-07 08:42:36.646241718 +0100
+++ b/package/firmware/linux-firmware/mediatek.mk	2025-03-07 08:43:40.000000000 +0100
@@ -88,14 +88,14 @@
 endef
 $(eval $(call BuildPackage,mt7986-wo-firmware))
 
-Package/mt7988-2p5g-phy-firmware = $(call Package/firmware-default,MT7988 built-in 2.5G Ethernet PHY firmware,,LICENCE.mediatek)
-define Package/mt7988-2p5g-phy-firmware/install
-	$(INSTALL_DIR) $(1)/lib/firmware/mediatek/mt7988
-	$(INSTALL_DATA) \
-		$(PKG_BUILD_DIR)/mediatek/mt7988/i2p5ge-phy-pmb.bin \
-		$(1)/lib/firmware/mediatek/mt7988
-endef
-$(eval $(call BuildPackage,mt7988-2p5g-phy-firmware))
+#Package/mt7988-2p5g-phy-firmware = $(call Package/firmware-default,MT7988 built-in 2.5G Ethernet PHY firmware,,LICENCE.mediatek)
+#define Package/mt7988-2p5g-phy-firmware/install
+#	$(INSTALL_DIR) $(1)/lib/firmware/mediatek/mt7988
+#	$(INSTALL_DATA) \
+#		$(PKG_BUILD_DIR)/mediatek/mt7988/i2p5ge-phy-pmb.bin \
+#		$(1)/lib/firmware/mediatek/mt7988
+#endef
+#$(eval $(call BuildPackage,mt7988-2p5g-phy-firmware))
 
 Package/mt7988-wo-firmware = $(call Package/firmware-default,MT7988 offload firmware,,LICENCE.mediatek)
 define Package/mt7988-wo-firmware/install
